incubator/hnc/internal/kubectl/tree.go

```diff
@@ -81,6 +81,21 @@ func printSubtree(prefix string, hier *api.HierarchyConfiguration, inCycle bool)
 		if cycle && inCycle {
 			continue
 		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		ns, err := k8sClient.CoreV1().Namespaces().Get(ctx, txt, metav1.GetOptions{})
+		if err != nil {
+			fmt.Printf("Could not list namespaces: %s\n", err)
+			os.Exit(1)
+		}
+
+		_, hasSubnamespaceAnnotation := ns.ObjectMeta.Annotations["hnc.x-k8s.io/subnamespaceOf"]
+		if hasSubnamespaceAnnotation {
+			txt = "[sub]" + txt
```

Comment:
Looking it over now, "[sub]" seems unclear.

@rjbez17 , wdyt? I'm in favour of either making it "[s] name" and have a "footer" showing what "[s]" means, or "[subns]" which is self-explanatory but longer.

Reply:
I think "[s]" + footnote may be slightly less clear but it will make viewing large trees much easier and frequent users will quickly understand what it means. But I could go either way really. I do agree "[sub]" is unclear.

Reply:
done